frontend/shared/src/components/forms/heading/Heading.tsx

```diff
@@ -21,7 +10,7 @@ const Heading: React.FC<HeadingProps> = ({
   loading,
   tooltip,
 }) => (
-  <$Header size={size || 'm'} as={as}>
+  <$Header size={size} as={as}>
```

Comment:
One last thing, if we are going to remove this default value, I suggest making the size a required prop.

Reply:
Reverting this too, or put it like `size = 'm',`